homeassistant/components/ialarmxr/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the iAlarmXR integration."""
+from pyialarmxr import IAlarmXR
+
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_TRIGGERED,
+)
+
+DEFAULT_PORT = 18034
+
+DEFAULT_HOST = "47.91.74.102"
```

Comment:
Is this IP for remote polling?

Reply:
HI @RyuzakiKK thank you for your review 🙌
Yes, it's the cloud host for the iAlarmXR service. Unfortunately, unlike what was done for iAlarm, for the XR alarm there no way to communicate with the alarm itself...very sad iAlarm architecture...anyway that's it 😞

Reply:
Oh, I didn't realize that.
Then probably you could just hardcode the port and the host? If you must use the iAlarm cloud, I guess that there isn't a point in giving the option to change them.

Reply:
Oh no, it's not hardcode. Could be changed in the configuration step adding this integration. As is are a default values as in the mobile app approach 👍

Reply:
<img width="406" alt="Screenshot 2022-04-22 at 12 33 11" src="https://user-images.githubusercontent.com/11453002/164700737-c88d3207-a892-4fd7-a485-acf2b878f89a.png">

Something like the mobile app:

![Screenshot_20220422_125046_com spialarmxr](https://user-images.githubusercontent.com/11453002/164701314-695fce46-6501-4a04-aeb3-5865f045edc1.jpg)

Reply:
Right. I was proposing to hardcode them if they were always the same for everyone.

But probably, as it is now, it's easier if ever iAlarm is going to edit their cloud IP?

So, fine by me to keep giving the option.

Reply:
Oh yes, or probably for some users it's pre-configured with an host and for other with another host...I don't know..boh! 😅
Thanks again for you review! 🙏